build/amis/ansible/roles/containerd/tasks/main.yml

```diff
@@ -32,18 +32,58 @@
   unarchive:
     remote_src: True
     src: /tmp/containerd.tar.gz
-    dest: /
+    owner: root
+    group: root
+    dest: /usr/local
     extra_opts:
       - --no-overwrite-dir
 
+- name: download crictl
+  get_url:
+    url: https://github.com/kubernetes-sigs/cri-tools/releases/download/v{{ cri_tools_version }}/crictl-v{{ cri_tools_version }}-linux-amd64.tar.gz
+    checksum: "sha256:{{ cri_tools_sha256 }}"
+    dest: /tmp/crictl.tar.gz
+
+- name: unpack crictl
+  unarchive:
+    remote_src: True
+    src: /tmp/crictl.tar.gz
+    mode: 0755
+    owner: root
+    group: root
+    dest: /usr/local/bin
+    extra_opts:
+      - --no-overwrite-dir
+
+- name: install systemd unit for containerd
+  copy:
+    src: "{{ role_path }}/files/etc/systemd/system/containerd.service"
```

Comment:
Does `etc/systemd/system/containerd.service` not work here?

Reply:
Actually, there shouldn't be a need to nest the `containerd.service` file so deeply under `files`, could just put it directly under `files`

Reply:
I can handle that as a followup.

Reply:
Happy for you to just change it before finalising the upstream PR. This is just to your PR.